narwhals/expression.py

```diff
@@ -1519,6 +1519,49 @@ def tail(self, n: int = 10) -> Expr:
 
         return self.__class__(lambda plx: self._call(plx).tail(n))
 
+    def round(self, decimals: int = 0) -> Expr:
+        r"""
+        Round underlying floating point data by `decimals` digits.
```

Comment:
i think pandas and polars may differ in how they round from midway (`.5`, `1.5`), shall we add a note about it?

Reply:
Do you know if the difference only applies in the range `(-.5, .5)`? 

> shall we add a note about it?
If we know how to correct it manually, I think we should. Steering pandas behavior to polars' one

Reply:
i think pandas rounds to the nearest even, polars towards zero? something like that

not sure about how to correct it reliably - maybe we can just document it for now?

Reply:
I added a note, but I would rather have a strategy to address cases like this one in the future 🤔